api/v1alpha4/metal3datatemplate_types.go

```diff
@@ -252,20 +255,30 @@ type NetworkDataLink struct {
 
 // NetworkDataService represents a service object
 type NetworkDataService struct {
+
 	// DNS is a list of DNS services
 	DNS []ipamv1.IPAddressStr `json:"dns,omitempty"`
+
+	//DNSFromIPPool is the name of the pool from which to get the DNS servers
```

Comment:
Since the purpose of a pool is known, replace every occurrence of:
"pool from which to get the" by "pool of"

Reply:
this would not be correct. I can change "pool" for "IPPool" maybe ?

Reply:
ok. that is good.